pkgs/pkgs-lib/formats.nix

```diff
@@ -1125,4 +1125,29 @@ optionalAttrs allowAliases aliases
     else
       throw "pkgs.formats.xml: Unknown format: ${format}";
 
+  plist =
```

Comment:
based on `plist` i fail to think of `PropertyList`. There are too many thinks that could us the `p` as a shortname. can we rename this to `propertyList` instead?

Reply:
or maybe its just my lack of knowledge... https://github.com/NixOS/nixpkgs/blob/5175ca40f3d5dfc972f8bbb16ec2a7f0195c3374/lib/generators.nix#L627